scripts/start-server.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
 # Pull the docker image with the latest changes on Keycloak
-docker pull jboss/keycloak:master
+docker pull jboss/keycloak:latest
```

Comment:
@mhajas because you touched this file, I have to ask. What do you think about using quay.io instead? In this way we keep consistency across our repos.

Reply:
Well I can't see any reason why not to use quay. I will try to add it and see whether Travis passes.